source/hardware/controllers/denon_mc7000.rst

```diff
@@ -21,51 +10,43 @@ analogue (vinyl) and digital sources.
 
 .. versionadded:: 2.2.4
 
-Features
-~~~~~~~~
+Initial mapping release
+
+
+.. versionchanged:: 2.3.0
+
+Fixed a bug for Vinylmode on Deck 4 and added new features, like library sorting, search through track with Jog Wheel, eject track from Deck, added waveform zoom, changed Parameter and Censor/Reverse/Spinback button mapping
```

Comment:
Markup is still broken. This needs to be indented by 3 spaces and without a blank line between `versionadded` and the text block. I'll just merge this and fix this myself :D

Reply:
sorry for this - obviously i didnt understand how you wanna have this formatted